mappings/net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5163 net/minecraft/world/gen/surfacebuilder/BasaltDeltasSurfaceBuilder
+	FIELD field_23915 DELTA_BASALT Lnet/minecraft/class_2680;
+	FIELD field_23916 DELTA_BLACKSTONE Lnet/minecraft/class_2680;
+	FIELD field_23917 DELTA_GRAVEL Lnet/minecraft/class_2680;
```

Comment:
```suggestion
	FIELD field_23915 BASALT Lnet/minecraft/class_2680;
	FIELD field_23916 BLACKSTONE Lnet/minecraft/class_2680;
	FIELD field_23917 GRAVEL Lnet/minecraft/class_2680;
```

Reply:
`GRAVEL`, `BASALT` and `BLACKSTONE` already exist in a superclass as final, thus these need unique names.